src/sentry/static/sentry/app/components/dropdownMenu.jsx

```diff
@@ -187,16 +221,22 @@ class DropdownMenu extends React.Component {
           onMouseEnter(...args);
         }
 
+        // Only handle mouse enter for nested dropdowns
         if (!isNestedDropdown) return;
-        this.handleOpen(...args);
+
+        if (this.mouseLeaveId) window.clearTimeout(this.mouseLeaveId);
+
+        this.mouseEnterId = window.setTimeout(() => {
```

Comment:
is there an easy way to know if no sibling menu is open? (you _could_ open immediately in that case)

Reply:
I don't think there's an easy way, I'm open to ideas if you have any